### ISSUE TITLE
Referenced templates do not create self-IPs or HA configurations

### ISSUE THREAD

**Issue by C0missar** [F5Networks/f5-aws-cloudformation#67]
## Do you already have an issue opened with F5 support?
Github Issues are consistently monitored by F5 staff, but should be considered as best effort only and you should not expect to receive the same level of response as provided by F5 Support. Please [open an case](https://support.f5.com/csp/article/K2633) with F5 if this is a critical issue.

## Description
Describe the problem you're having or the enhancement you'd like to request.
```
The templates output self-IPs for the public/external and private/internal interfaces, but do not actually create them in the config.
```
```
The device pairs come up standalone, not HA (this may be related to issue #63).
```
```
There is no password set for admin, nor is there a "quickstart" user with password = AWS Instance ID as described on the Quick Start page.
```
```
The license keys entered during the stack build are not assigned to the images.  When executing the "Manual license" option, they must be re-entered.
```
## Template
For bugs, enter the template with which you are experiencing issues below.
```
f5-existing-stack-across-az-cluster-byol-3nic-bigip.template
```
## Severity Level
For bugs, enter the bug severity level. Do not set any labels.

Severity: <Fill in level: 1 through 5>
1
Severity level definitions:
 1. **Severity 1 (Critical)** : Defect is causing systems to be offline and/or nonfunctional. immediate attention is required.
 2. **Severity 2 (High)** : Defect is causing major obstruction of system operations.
 3. **Severity 3 (Medium)** : Defect is causing intermittent errors in system operations.
 4. **Severity 4 (Low)** : Defect is causing infrequent interuptions in system operations.
 5. **Severity 5 (Trival)** : Defect is not causing any interuptions to system operations, but none-the-less is a bug.

**Comment by edwoodjrjr** [comment]
Yes, it does sound a lot like issue 63, which will be fixed in an imminent release.

Can you provide the logs from /var/log/cloud/aws/?

**Comment by C0missar** [comment]
There is no /var/log/cloud directory on any of the AWS builds.

Stan Ward
IT Application Security Architect
The Sherwin-Williams Company
Cell: 937.344.8342
Stan.Ward@Sherwin.com
http://www.sherwin-williams.com


-----"Mike Shimkus" <notifications@github.com> wrote: -----
To: "F5Networks/f5-aws-cloudformation" <f5-aws-cloudformation@noreply.github.com>

**Comment by edwoodjrjr** [comment]
What do you have in /config/cloud?

**Comment by C0missar** [comment]
That directory doesn't exist, either.

Stan Ward
IT Application Security Architect
The Sherwin-Williams Company
Cell: 937.344.8342
Stan.Ward@Sherwin.com
http://www.sherwin-williams.com


-----"Mike Shimkus" <notifications@github.com> wrote: -----
To: "F5Networks/f5-aws-cloudformation" <f5-aws-cloudformation@noreply.github.com>

**Comment by edwoodjrjr** [comment]
Then that will be difficult to troubleshoot.  I recommend you open a support case with F5.  They can escalate to the cloud solutions team if necessary.

**Comment by edwoodjrjr** [comment]
That's unfortunate, but the escalation process has been revisited since then and should be understood by all involved now.

We are tracking issue #60 with internal ID #895.  When that is released we'll make a note here to let you know. I don't have an ETA on it yet.

**Comment by C0missar** [comment]
Ok, and thanks for checking.  I have a CFT that I think resolves #60 (although it should become a supported template), but it has the same problems as issue #67.

Cheers,
Stan

**Comment by crosbygw** [comment]
Issue resolved in release [v4.1.2](https://github.com/F5Networks/f5-aws-cloudformation/releases/tag/v4.1.2)
